src/nunit.analyzers/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -78,7 +78,7 @@ public static string GetName(this ExpressionSyntax expression)
                 case IdentifierNameSyntax identifierName:
                     return identifierName.Identifier.Text;
                 default:
-                    return null;
+                    return "";
```

Comment:
Why isn't it better for the caller to explicitly handle the possibility that there was no name?

Reply:
I suppose you're right, will submit separate PR.